x-pack/test/upgrade_assistant_integration/config.js

```diff
@@ -9,6 +9,9 @@ import {
   EsProvider,
 } from './services';
 
+// Temporary until https://github.com/elastic/kibana/pull/29184 is merged
+delete process.env.TEST_ES_SNAPSHOT_VERSION;
```

Comment:
Is this necessary?

Reply:
Also that PR mentioned is long since merged.

Reply:
https://github.com/elastic/kibana/pull/32580/commits/68858620736b482fcb00cf1058f8ed61fb5c0371